### ISSUE TITLE
Question: Easy way to read zcount along with zgrad for postprocessing?

### ISSUE THREAD

**Issue by ReneHamburger1993** [Colvars/colvars#448]
Hi, 

I am trying to figure out an easy way to call czar estimator using the c++ code.

Poisson-Integration  can be done easily by reading a gradient file as shown in [https://github.com/Colvars/colvars/blob/13fc2238347987335654c20313366ab19fd21fa9/colvartools/poisson_integrator.cpp](url)

but how to load a zcount file along to a zgrad file it?

For me it seems I cannot initialize a count grid properly without reading a colvar config as well (while this is possible for the gradients), is that correct?

Kind regards
René

**Comment by jhenin** [comment]
Hi René,

You could read in a count file using a very similar mechanism to that used for the gradient, but then to run the CZAR estimator, you are going to need an ABF bias, which will want colvar definitions. Maybe the simplest way to implement this would be to create dummy collective variables based on grid files, and that would not be very simple. Maybe there is a simpler way but I can't see it right now.

Best,
Jérôme

**Comment by ReneHamburger1993** [comment]
Hi Jérôme,

sorry for not responding earlier. Just got a hand on this yesterday again.
Though it was not *simple* (as I had to add another constructor for the colvar_grid_gradient)
but I managed to get it working (without reading any colvar definitions) such that I can read zgrad/zcount file(s) that are merged properly and czar estimator + integration is called.

I will share this code once I got my analysis done.

Best,
René
